pilot/pkg/bootstrap/server.go

```diff
@@ -282,6 +286,33 @@ func (s *Server) initClusterRegistries(args *PilotArgs) (err error) {
 	return err
 }
 
+// GetMeshConfig fetches the ProxyMesh configuration from Kubernetes ConfigMap.
+func GetMeshConfig(kube kubernetes.Interface, namespace, name string) (*v1.ConfigMap, *meshconfig.MeshConfig, error) {
```

Comment:
cool,so you can also remove
https://github.com/istio/istio/blob/master/tests/e2e/tests/pilot/util/environment.go#L98
and
https://github.com/istio/istio/blob/master/tests/e2e/tests/pilot/util/environment.go#L149
?